pages/common/scrapy.md

```diff
@@ -0,0 +1,31 @@
+# scrapy
+
+> Open Source framework for crawling websites written in python.
+
+- Create a project:
+
+`scrapy startproject {{project_name}}`
+
+- Create a spider:
+
+`scrapy genspider {{spider_name}} {{website_domain}}`
+
+- Edit spider:
+
+`scrapy edit {{spider_name}}`
+
+- Run spider:
+
+`scrapy crawl {{spider_name}}`
+
+- Fetch a webpage as scrapy sees it and print source in stdout:
+
+`scrapy fetch {{url}}`
```

Comment:
Is this example (and the following ones) meant to be executed within a project, or can they be executed as one-off commands?